contracts/seed/Seed.sol

```diff
@@ -53,25 +53,30 @@ contract Seed {
     // Contract logic
     bool      public closed;
     bool      public paused;
-    uint256   public totalLockCount;
+    uint256   public totalLockCount;   // Total locks that have been created. Each user can have only one lock.
+    uint256   public seedRemainder;    // Amount of seed tokens remaining to be distributed
+    uint256   public seedClaimed;      // Amount of seed token claimed by the user.
+    uint256   public fundingCollected; // Amount of funding tokens collected by the seed contract.
+    uint256   public fundingWithdrawn; // Amount of funding token withdrawn from the seed contract.
     bool      public initialized;
     bool      public minimumReached;
     bool      public maximumReached;   
 
     mapping (address => bool)    public whitelisted;
     mapping (address => Lock)    public tokenLocks; // locker to lock
 
-    event LockAdded(address indexed recipient, uint256 locked);
-    event TokensClaimed(address indexed recipient, uint256 amountVested);
+    event SeedsPurchased(address indexed recipient, uint256 locked);
+    event TokensClaimed(address indexed recipient,uint256 amountVested,address indexed beneficiary,uint256 feeAmount);
```

Comment:
instead of "amountVested", can we just call this "amount"?

Reply:
Okay